Devices/igtlioVideoDevice.cxx

```diff
@@ -140,27 +142,42 @@ igtl::MessageBase::Pointer VideoDevice::GetIGTLMessage()
   float bitRatePercent = 0.05;
   int frameRate = 20;
   int iReturn = 0;
-#ifdef OpenIGTLink_LINK_VP9
-  if(this->useVP9)
+  this->OutVideoMessage = igtl::VideoMessage::New();
+#if defined(OpenIGTLink_USE_H264)
+  if(this->CurrentCodecType.compare(IGTL_VIDEO_CODEC_NAME_H264) == 0)
```

Comment:
Do these codec names allow for different implementations?

Reply:
Unfortunately not, as you are implementing the intel h264 decoder. 
You could probably add another constant as: IGTL_VIDEO_CODEC_NAME_INTEL_H264.
Or any better suggestions?